Objects/genericaliasobject.c

```diff
@@ -398,6 +398,8 @@ ga_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
 PyTypeObject Py_GenericAliasType = {
     PyVarObject_HEAD_INIT(&PyType_Type, 0)
     .tp_name = "types.GenericAlias",
+    .tp_doc = "A type to represent the type 'origin' as generic with respect to the type "
+              "arguments 'args'. See PEP 585 for more.",
```

Comment:
May I suggest a somewhat different text? How about:

"""
Represent a PEP 585 generic type

E.g. for t = list[int], t.origin is list and t.args is (int,).
"""